java/idl2jni/codegen/im_jni.cpp

```diff
@@ -1649,8 +1659,10 @@ ostream &operator<< (ostream &o, AST_Expression::AST_ExprValue *ev)
   case AST_Expression::EV_wchar:
     o << "L\'" << ev->u.wcval << '\'';
     break;
+  case AST_Expression::EV_int8:
```

Comment:
I think combining `uint8` and `octet` are okay, but the `int8` values won't come out right because the other two are unsigned. This is what I'm doing in `dds_generator.h`: https://github.com/objectcomputing/OpenDDS/blob/ef88d4e8561b8e3bbad42b953118949a2322f6b2/dds/idl/dds_generator.h#L684-L685